batcher/batcher/cardano_chain_operations_strategy.go

```diff
@@ -324,6 +344,21 @@ func getNeededUtxos(
 	return chosenUTXOs, nil
 }
 
+func getKnownTokens(cardanoConfig *cardano.CardanoChainConfig) ([]cardanowallet.Token, error) {
```

Comment:
maybe this function should be util function putted somewhere where other similar are placed because probably someone else will need this func too